aksel.nav.no/website/app/dev/produktbloggen/_ui/Produktbloggen.module.css

```diff
@@ -1,9 +1,125 @@
+.overviewTitle {
+  font-size: 48px;
+  color: var(--ax-text-brand-blue);
+  text-align: center;
+}
+
+.overviewSubtitle {
+  color: var(--ax-text-brand-blue-subtle);
+  text-align: center;
+  max-width: 670px;
+  font-size: var(--ax-font-size-heading-small);
+}
+
+.latestBloggPosts {
+  display: grid;
+  gap: var(--ax-space-48);
+  margin-inline: auto;
+  margin-block: var(--ax-space-48);
+}
+
+.bloggPosts {
+  margin-inline: auto;
+  margin-bottom: var(--ax-space-128);
+  display: grid;
+  width: 100%;
+  padding-inline: var(--ax-space-16);
+  max-width: var(--ax-breakpoint-2xl);
+}
+
+.remainingPosts {
+  display: grid;
+  gap: var(--ax-space-48);
+  margin-top: var(--ax-space-8);
+}
+
+.article {
+  grid-column: span 1 / span 1;
+}
+
+.articleBody {
+  margin-top: var(--ax-space-16);
+}
+
+.articleBodySubtle {
+  margin-top: var(--ax-space-16);
+  color: var(--ax-text-neutral-subtle);
+}
+
+.articleAuthor {
+  display: flex;
+  margin-top: var(--ax-space-16);
+  gap: var(--ax-space-8);
+  color: var(--ax-text-neutral-subtle);
+
+  & span:first-child {
```

Comment:
Not a huge fan of first-child "as special" selectors, since they can be a bit fragile. But kinda balances more classNames={module.thing} "bloat" (not really bloat per se, but it feels like a lot to write for maybe 1 css property to be applied) in the jsx with more cryptic and fragile CSS 🤔 . Is it fine here? or should we strictly have _everything_ gets a css module on first class basis?

Reply:
Agreed that it often feels a little cumbersome to add more declarative CSS for cases like this. In cases like this we can solve it by using our components like this 
```diff
- <BodyShort size="small" className={styles.articleAuthor}>
-   <span>{getAuthors(blogg)[0]}</span>
-   <span>{date}</span>
- </BodyShort>
+ <BodyShort size="small" className={styles.articleAuthor}>
+   <BodyShort as="span" size="small" weight="semibold">{getAuthors(blogg)[0]}</BodyShort>
+   <span>{date}</span>
+ </BodyShort>
```